pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -586,7 +589,63 @@ func TestZeroHintMath(t *testing.T) {
 				}),
 			},
 		},
-
+		"SignedDivRem": {
+			{
+				operanders: []*hintOperander{
+					{Name: "value", Kind: apRelative, Value: feltString("0")},
+					{Name: "div", Kind: apRelative, Value: &utils.FeltMax128},
+					{Name: "bound", Kind: apRelative, Value: felt127},
+					{Name: "r", Kind: reference, Value: addrBuiltin(starknet.RangeCheck, 0)},
+					{Name: "biased_q", Kind: reference, Value: addrBuiltin(starknet.RangeCheck, 1)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newSignedDivRemHint(ctx.operanders["value"], ctx.operanders["div"], ctx.operanders["bound"], ctx.operanders["r"], ctx.operanders["biased_q"])
+				},
+				errCheck: errorTextContains("div="),
```

Comment:
Probably better to check the exact error string 🤔 
Same for other `errCheck` below.

Reply:
done